pom.xml

```diff
@@ -39,39 +39,45 @@
     </license>
   </licenses>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.jenkins.tools.bom</groupId>
+        <artifactId>bom-${jenkins.baseline}.x</artifactId>
+        <version>4740.v75a_90f6fefb_7</version>
+        <scope>import</scope>
+        <type>pom</type>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
```

Comment:
The bom for the current baseline containing most (important) plugins, making manually updating versions of plugins etc. easier and less error prone.

https://www.jenkins.io/doc/developer/plugin-development/dependency-management/